### ISSUE TITLE
New emerging VoC in 2025

### ISSUE THREAD

**Issue by yangclaraliu** [jarawees/covidmodel-hitap#31]
Can you tell me a little bit more about this variant emerging on 2025-01-01?
https://github.com/jarawees/covidmodel-hitap/blob/c00834070a8c3d0dff7c9a57414b71a4dd107fce/code/0_1a_util_HITAP.R#L619

**Comment by SiobhanSiobhan** [comment]
We put in an extra variant in future in case we want to look at scenarios that would make our most cost-effective option not the most cost-effective. Noted that we need to update the rest of the code!

**Comment by yangclaraliu** [comment]
yes but this variant, from the parameter, looks like it's largely modelled against Omicron (in terms of VE for instance) so it'd look like you are introducing omicron in 2025?

**Comment by SiobhanSiobhan** [comment]
it's a placeholder for now :)

**Comment by yangclaraliu** [comment]
Yes but it does partially lead to your astronomical number in 2030 that Jar has pointed out (i.e. really large numbers after 2025). It might not be the whole cause but I'm still checking.

**Comment by yangclaraliu** [comment]
At the end of the day, it is the drastic change in ve that led to the astronomical numbers you are seeing, with everyone in the population getting multiple infections each year. With the current implementation, the VE change, also doesn't affect never-vaccinated individuals because it's attached to vaccines. This means vaccinated individuals may perform worse than never-vaccinated individuals when you implement really really low protection. It's sort of equivalent to letting COVID-19 burn through the population naturally and assuming very little protection from even infection.

I do think it's fair to say if your analytical window is on the decadal level that no government/ WHO will keep rolling out a vaccine with protection <30%. 

Another reason for steady increase in severe outcomes is the aging population. According to WPP, the population above 75 will increase 30% by 2025 and ~70% by 2030. In the model right now, we are implementing a non-time-varying mortality rate . this did lead to a significant increase in 75+ pop which led to an increase in pop experiencing high susceptibility against SARS-CoV-2 and high IFR/ IHR. 

I have put some of the codes I've been playing within a new code file called `6_debug.R` and suggest that both @SiobhanSiobhan and @jarawees play around with these different ways to examine the results and see if they make sense. At the end of the day, this model was fitted to six months of death data, and the ihr and picu are not local so I don't expect the predicted hospitalisation to match up with observed hospitalisation exactly.

**Comment by SiobhanSiobhan** [comment]
Thank you Yang!!! OK we will take a look at that file
But we are very happy to now have normal looking deaths!
